### DISCUSSION TITLE
Spotify was uninstalled - intentional or bug?

### DISCUSSION THREAD

**Discussion by Southclaws** [Raphire/Win11Debloat#110] [Q&A]
Is Spotify meant to be uninstalled by this script, I thought it was just focused on "bloatware" from Microsoft? I assumed it was because of explorer.exe refresh or something my music stopped.

**Comment by Raphire** [comment]
Spotify used to be one of the third-party apps that came preinstalled with Windows, which is the reason it is included the script. 

I am unsure if this is still the case however, I don't think I've seen it included recently outside of a stub in the start menu. 

I'll look into removing popular apps like Spotify from the removal list.

**Reply by Southclaws** [reply]
Ah yeah, I wasn't aware it was pre-installed, I definitely installed it myself so I was surprised a user-installed app was removed by the script. If it's helpful, I set up my machine in 2021 and Spotify was not pre-installed then, I installed it myself from the Microsoft Store.

Thanks for the clarification!

**Reply by Southclaws** [reply]
Perhaps it's possible to detect whether an app was pre-installed or user-installed via MS store APIs or inferring it from location or registry entries?

**Reply by evrial** [reply]
> I'll look into removing popular apps like Spotify from the removal list.


so any success?
thanks to you I have to re download my music library

**Reply by Raphire** [reply]
@evrial Sorry you had to redownload your music. As or release 2025.09.13 it's possible to customize what apps are removed as part of the default mode.

**Reply by evrial** [reply]
thanks!
